src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -60,27 +64,30 @@ object JesBackend {
 
   def stderrJesOutput(callGcsPath: String): JesOutput = JesOutput(LocalStderrParamName, s"$callGcsPath/$LocalStderrValue", Paths.get(LocalStderrValue))
   def stdoutJesOutput(callGcsPath: String): JesOutput = JesOutput(LocalStdoutParamName, s"$callGcsPath/$LocalStdoutValue", Paths.get(LocalStdoutValue))
-  def localizationDiskInput(): JesInput = JesInput("working_disk", "disk://local-disk", new File(JesCromwellRoot).toPath)
+  def localizationDiskInput(): JesInput = JesInput(WorkingDiskParamName, LocalWorkingDiskValue, new File(JesCromwellRoot).toPath)
+  def authGcsCredentialsPath(gcsPath: Option[String]): Option[JesInput] =
+    gcsPath.flatMap(x => Some(JesInput(ExtraConfigParamName, x, Paths.get(""), "LITERAL")))
```

Comment:
When creating an `Option` literal it's a good habit to use `Option(...)` over `Some(...)` to handle nulls correctly, even though here the `...` expression obviously can't be null.